libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -377,17 +377,16 @@ public static boolean smartStoreExists(Context ctx, String dbNamePrefix,
 
 	static class DBHook implements SQLiteDatabaseHook {
 		public void preKey(SQLiteDatabase database) {
-			// Using sqlcipher 2.x kdf iter because 3.x default (64000) and 4.x default (256000) are too slow
-			// => should open 2.x databases without any migration
-			database.execSQL("PRAGMA cipher_default_kdf_iter = 4000");
 		}
 
 		/**
 		 * Need to migrate for SqlCipher 4.x
 		 * @param database db being processed
 		 */
 		public void postKey(SQLiteDatabase database) {
-			database.rawExecSQL("PRAGMA cipher_migrate");
```

Comment:
We are not doing cipher_migrate on iOS anymore (https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SmartStore/SmartStore/Classes/SFSmartStoreDatabaseManager.m#L208).